src/gatsby-node.js

```diff
@@ -3,9 +3,14 @@ import LoadablePlugin from '@loadable/webpack-plugin';
 import { statsFilename, statsPath } from './constants';
 
 export const onCreateWebpackConfig = ({ actions, stage }) => {
-  if (stage === 'build-javascript') {
+  if (stage === "build-javascript" || stage === "develop") {
```

Comment:
Adding ` || stage === 'develop-html'` here fixes a missing file error when running `gatsby develop` on Gatsby `3.3.0`

Reply:
Hi all, do you have any plans to fix this ?

Reply:
@drskigen I've pushed the change I'm using locally onto this branch. It should solve the issue with 3.2+.

@PetroPavlenko I'm not sure if you mean this specific change which is now updated or with the release version of this plugin in general. It doesn't look like the plugin is maintained, so I don't hold out much hope of it ever being updated.